lib/lutaml/model/xml_mapping.rb

```diff
@@ -1,4 +1,5 @@
 require_relative "xml_mapping_rule"
+require "securerandom"
```

Comment:
@0arm0 why do we need to use `SecureRandom`? This is an unnecessary dependency. I don't see why we can't number the names sequentially.